docs/mddocs/Quickstart/llama_cpp_quickstart.md

```diff
@@ -316,3 +316,9 @@ Also, you can use `ONEAPI_DEVICE_SELECTOR=level_zero:[gpu_id]` to select device
 If you run the llama.cpp program on Windows and find that your program crashes or outputs abnormally when accepting Chinese prompts, you can open `Region->Administrative->Change System locale..`, check `Beta: Use Unicode UTF-8 for worldwide language support` option and then restart your computer.
 
 For detailed instructions on how to do this, see [this issue](https://github.com/intel-analytics/ipex-llm/issues/10989#issuecomment-2105600469).
+
+#### Sycl7.dll not found error
+If you meet `System Error: sycl7.dll not found`, please check if you are using a conda environment and in the right oneapi env, or if you executed `source /opt/intel/oneapi/setvars.sh` on Linux
+
+#### Garbage output
+If you meet garbage output, please check if your GPU driver version is >= [31.0.101.5522](https://www.intel.cn/content/www/cn/zh/download/785597/823163/intel-arc-iris-xe-graphics-windows.html)
```

Comment:
If you meet garbage output, please check first if your GPU driver version is >= [31.0.101.5522](https://www.intel.cn/content/www/cn/zh/download/785597/823163/intel-arc-iris-xe-graphics-windows.html). If not, please follow the instructions in [this section](./install_windows_gpu.md#optional-update-gpu-driver) to update your GPU driver.